features/step_definitions/tax_relief_warning_steps.rb

```diff
@@ -0,0 +1,32 @@
+Given(/^that I am on the WPCC homepage in my own "([^"]*)"$/) do |language|
+  locale = language_to_locale(language)
+
+  your_details_page.load(locale: locale)
+end
+
+When(/^I enter my personal details$/) do
+  your_details_page.age.set(35)
+  your_details_page.genders.select(
+    I18n.translate('wpcc.details.options.gender.female')
+  )
+  your_details_page.salary_frequencies.select(
+    I18n.translate('wpcc.details.options.salary_frequency.year')
+  )
+  your_details_page.minimum_contribution_button.set(true)
+end
+
+When(/^I enter a "([^"]*)" below the salary band$/) do |salary|
+  your_details_page.salary.set(salary)
+end
+
+When(/^I progress to the results page$/) do
+  step 'I move to your results page'
+end
+
+Then(/^I should see tax relief "([^"]*)"$/) do |warning_message|
+  expect(page).to have_content(warning_message)
+end
+
+def language_to_locale(language)
+  { 'English' => :en, 'Welsh' => :cy }[language]
+end
```

Comment:
It is not a good practice to add global methods, and mainly in isolation. 

It is better we move into a World module now, because the code is adding this method to the main context. Basically if I create the same method name in a World module, it could cause clashing colisions when loading this steps, and **will be very hard to investigate the reasons** since the method is on a particular step definitions. 

And if someone see this pattern and start to do the same (broken windows) then it will become more difficult to maintain either.

And will be hard because maybe the developer could not know where this method comes from, which then the developer should be doing something like `method(:language_to_locale).owner` to discover that. Which requires more knowledge of Ruby internals.

So conclusion, it is not a good practice to add global methods everywhere. That's the reason Cucumber has the World clause and Rspec has the config.include calls.